schema/article.js

```diff
@@ -22,6 +24,13 @@ const ArticleType = new GraphQLObjectType({
       type: AuthorType,
       resolve: ({ author }) => author,
     },
+    channel_id: {
+      type: GraphQLString,
+    },
+    contributing_authors: {
+      type: new GraphQLList(AuthorType),
+      resolve: ({ contributing_authors }) => contributing_authors,
+    },
```

Comment:
Does this look ok locally? `contributing_authors` is an array of author-like objects